CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (crypto | x/auth) [#14372](https://github.com/cosmos/cosmos-sdk/pull/18194) Key checks on signatures antehandle.
 * (staking) [#18506](https://github.com/cosmos/cosmos-sdk/pull/18506) Detect the length of the ed25519 pubkey in CreateValidator to prevent panic.
 * (types) [#18372](https://github.com/cosmos/cosmos-sdk/pull/18372) Removed global configuration for coin type and purpose. Setters and getters should be removed and access directly to defined types.
+* (types) [#18695](https://github.com/cosmos/cosmos-sdk/pull/18695) Removed global configuration for txEncoder.
```

Comment:
Can we always precise the alternative when we remove something from the config?

Maybe it would be nice to add an UPGRADING.md section on the global config removals, so there will be one place to look at it.

Reply:
This one has not been used anywhere, I think it is deprecated, thats why I didn't add an alternative, TxEncoder is used on other parts but not the one dfined as a global config. I dont see any use cases where someone would want some sort of global txEncoder

Reply:
A quick search here: https://sourcegraph.com/search?q=context%3Aglobal+.GetTxEncoder%28%29&patternType=standard&sm=1&groupBy=repo shows indeed that it is only used in v0.45 chains / fork. Then I guess it is fine indeed.